### DISCUSSION TITLE
Subject Line Bug - ChatGPT-Writer adds the PROMPT to the Subject in email replies

### DISCUSSION THREAD

**Discussion by wittier** [GorvGoyl/ChatGPT-Writer#84] [Report Bugs and Issues]
I like how this writes email replies but there is a major bug. 
I am on a Windows 10 PC with Google Chrome browser running GMAIL in the browser. 
When I compose and send a reply, the extension modifies the email subject line and puts the GPT PROMPT into the subject.

 This is one long example of the subject line that it sends: 

_> Only append the subject text, don't add a prefix to the subject. Reply in the same language. The email length should be Brief. The email style should be Casual. The email urgency should be Non-urgent. The email tone should be Friendly. The email signature should be from: Ken DANIELI The email should be addressed to the following people: Fatma Hadi Reply to the following email: Fatma Haxxxdi <fhadi@xxx.com>Apr 6, 2023, 11:50 AM (20 hours ago)to ken@xxx.com, AgisilaosHi Ken, I hope this email finds you well. I am reaching out regarding the consultation you have scheduled with our client on industry perspectives for product channel strategy. Our client has shared additional material for this consultation for you to familiarize yourself with briefly over 5-10 minutes before your call on Tuesday. I have attached the relevant material here for your review. Please let me know of the safe receipt of the docume_ 

Here is another one:  

_> Only append the subject text, don't add a prefix to the subject. Reply in the same language. The email length should be Brief. The email style should be Casual. The email urgency should be Non-urgent. The email tone should be Friendly. The email signature should be from: Ken DANIELI The email should be addressed to the following people: Nicholxxxxayat Reply to the following email: Ken Danieli <ken@xxxxx.com>Fri, Mar 31, 8:14 AM (10 days ago)to NicholasGreat, thanks for the update, Nicholas.Best regards, Ken_ 

So this is a pretty bad bug

**Comment by GorvGoyl** [comment]
I don't fully understand the issue here. could you share some screenshot of
how it looks?

**Comment by wittier** [comment]
I had replied to Fatma using ChatGPT-Writer.  When Fatma replied to me,
this is what I see.

The email subject contains the prompt from the extension. "Re: Only
append...."

[image: image.png]

**Comment by GorvGoyl** [comment]
is there any possibility that you or Fatma is using some other email writing assistant that is conflicting with ChatGPT writer? Could you try replying to some other person to see if you still face the issue

**Comment by wittier** [comment]
Thanks for responding.

This is what the extension looks like when embedded in my Gmail
[image: image.png]

I do not think it happens every time I use ChatGPT-Writer to reply.  But it
happens with some frequency.
I do not think that the other person receiving the email is a factor.

Here is a screenshot of my inbox when I search for "append the subject,"
which is not language that I use. it's PROMPT language, related to creating
an email via AI.

[image: image.png]

I am not sure if another extension in my Chrome is interacting or not.  I
believe that I am using ChatGPT-Writer extension.
See what it looks like at the top of this email reply above.  Can you
confirm that what ChatGPT-Writer looks like when running in Gmail in Chrome?
I don't think other extensions are writing extensions.

When I use the extension to help write a reply, I then see a separate email
thread often in my inbox. So the reply is sometimes not included in the
original email thread.
In other cases I think that I've seen the Extension create a new subject
for the reply in a new thread along the lines of "RE: Your email from April
7."   I did not write that new subject.  The AI wrote it and created a new
email thread rather than a reply in the original email thread.  The lower
email is the original. Today I used the Extension to reply to the
original.  In my Inbox, it creates a new thread with my name as the sender
with an AI-created new subject that I did not write.   It is not useful to
have replies create new email threads.  Replies should remain in the
original email thread.  Screenshot of this:

[image: image.png]

Thanks for your help. I'd like to figure this out.

**Comment by GorvGoyl** [comment]
I can't see attached images.

**Comment by wittier** [comment]
I am replying to emails with images and the images are in my sent emails.
Github is suppressing the images in the responses when Github posts my
replies to the site.  I am not proficient at github. Is there  a way for me
to post the images or get the to you?

Thanks.

**Comment by GorvGoyl** [comment]
mail me at hey@gourav.io

**Reply by wittier** [reply]
> mail me at [hey@gourav.io](mailto:hey@gourav.io)

Thanks. I forwarded the emails with the screenshots to your email.
